README.md

```diff
@@ -6,14 +6,13 @@ The browser SDK is used to collect logs and RUM data from the browser.
 
 This repository contains several packages:
 
-| Package              | npm                      | size                     | cdn                        | doc                                 |
-| -------------------- | ------------------------ | ------------------------ | -------------------------- | ----------------------------------- |
-| browser-logs         | [![npm version][01]][02] | [![bundle size][03]][04] | [datadog-logs][05]         | [![API][1]][07] [![product][2]][08] |
-| browser-rum          | [![npm version][11]][12] | [![bundle size][13]][14] | [datadog-rum][15]          | [![API][1]][17] [![product][2]][18] |
-| browser-rum-slim     | [![npm version][21]][22] | [![bundle size][23]][24] | [datadog-rum-slim][25]     | [![API][1]][27] [![product][2]][28] |
-| browser-rum-recorder | [![npm version][31]][32] | [![bundle size][33]][34] | [datadog-rum-recorder][35] | [![API][1]][37] [![product][2]][38] |
-| browser-rum-core     | [![npm version][51]][52] | [![bundle size][53]][54] |                            |
-| browser-core         | [![npm version][41]][42] | [![bundle size][43]][44] |                            |
+| Package          | npm                      | size                     | cdn                    | doc                                 |
+| ---------------- | ------------------------ | ------------------------ | ---------------------- | ----------------------------------- |
+| browser-logs     | [![npm version][01]][02] | [![bundle size][03]][04] | [datadog-logs][05]     | [![API][1]][07] [![product][2]][08] |
+| browser-rum      | [![npm version][11]][12] | [![bundle size][13]][14] | [datadog-rum][15]      | [![API][1]][17] [![product][2]][18] |
+| browser-rum-slim | [![npm version][21]][22] | [![bundle size][23]][24] | [datadog-rum-slim][25] | [![API][1]][27] [![product][2]][28] |
```

Comment:
bundle size image not working

Reply:
Thanks for the review! The package is not yet published to NPM, so this is expected. It should work once released/published.